### PR TITLE
Removed obsolte sendEmailToAdmins und _GAE_SendEmail functions

### PR DESCRIPTION
**Pull request by tsteinruecken**

### CODE REVIEW

File: dbtransfer.py [hunk 1]
```diff
@@ -417,11 +417,8 @@ def exportItems(module, target, importKey, startCursor, endCursor):
 								headers={'Content-Type': 'application/x-www-form-urlencoded'})
 
 	if startCursor == endCursor:
```
Comment (phorward) [inline dbtransfer.py:419]: Please check according to #51 if this test can fail now in ViUR3!

**Comment by tsteinruecken** [comment]
Regarding sendEmailToAdmins: I think the best way is to utilize Stackdriver - configure an email/sms alert when something with critical is logged. This will then also work if no email service is configured.

The Cursor-Thing is a different issue :)

Regarding #24 please see comments there

**Comment by sveneberth** [comment]
> Regarding #24 please see comments there

Which comments?

**Comment by tsteinruecken** [comment]
Superseded by #195 and #200

**Pull request closed without merging**